core/src/main/java/de/valtech/aecu/core/groovy/console/bindings/actions/resource/MoveResourceToPathRegex.java

```diff
@@ -35,19 +40,21 @@
 public class MoveResourceToPathRegex implements Action {
 
     private ResourceResolver resourceResolver;
+    private boolean dryRun;
     private String matchPattern;
     private String targetPathExpr;
 
     /**
      * Constructor
      * 
-     * @param matchPattern     regex pattern
-     * @param targetPathExpr   target regex
-     * @param resourceResolver resolver
+     * @param matchPattern   regex pattern
+     * @param targetPathExpr target regex
+     * @param context        binding context
      */
     public MoveResourceToPathRegex(@Nonnull String matchPattern, @Nonnull String targetPathExpr,
-            @Nonnull ResourceResolver resourceResolver) {
-        this.resourceResolver = resourceResolver;
+            @Nonnull BindingContext context) {
+        this.resourceResolver = context.getResolver();
+        dryRun = context.isDryRun();
```

Comment:
Keep the context reference as class attribute because dryRun is set later in the call stack, else dryRun will be always true.

Reply:
fixed